pyramid_swagger/exceptions.py

```diff
@@ -8,15 +8,21 @@
 
```

Comment:
Would you add a section (or new page) to the documentation about how to use this decorator in practice? It can be pretty briefly, just giving a small paragraph on the potential reason for it and an example or two.

I'd like to make sure we clarify what exceptions you can expect and a couple example context managers.